### DISCUSSION TITLE
Can't connect Garmin GlO2 with Android 13

### DISCUSSION THREAD

**Discussion by JMVNunes** [OpenOrienteering/mapper#2189] [Questions & Answers]
Hi

Anyone else have problems activating with on tablet Samsung S7 Fe (SM-T733), with external GPS Garmin GLO2

it will not recognize the location, even with tablet internal GPS.
and also when drawing menu button doesn´t expand.

It is with Android 13

Tks

**Comment by pkturner** [comment]
I use an older Samsung tablet with GLO2. You need to run "Bluetooth GPS" or a similar app to override the internal GPS.

**Reply by JMVNunes** [reply]
Hi, it recognize with App Bluetooth GPS, and choosing NMEA, but once open it doesn't regist the movement or the changes of position.

It might be something with OS Android?
